externals/tXml.js

```diff
@@ -51,21 +51,44 @@ export const XML_ENTITIES = {
 };
 
 /**
- * Translates XML entities to their respective characters.
+ * Translates XML entities and character references to their respective characters.
```

Comment:
```suggestion
 * Translates XML predefined entities and character references to their respective characters.
```

Reply:
The function is not limited to translating the XML predefined entities set. Other entities may be given to it via the `entitiesList`.

Reply:
That's true, but any other entities would need to be defined in the XML document, and there's nothing parsing out entity declarations and adding them to the `entitiesList`, so it really is limited to predefined entities and character references.